radicle-cli/examples/rad-patch.md

```diff
@@ -83,3 +83,27 @@ new file mode 100644
 index 0000000..e69de29
 
 ```
+
+Wait, lets add a README too! Just for fun.
+
+```
+$ touch README.md
+$ git add README.md
+$ git commit --message "add README for the YOLOs"
```

Comment:
```suggestion
$ git commit --message "Add README, just for fun"
```
I find the yolo a bit distracting in this context :)

Reply:
done :)